manifests/z/ZAP/ZAP/2.13.0/ZAP.ZAP.installer.yaml

```
@@ -0,0 +1,21 @@
+PackageIdentifier: ZAP.ZAP
+PackageVersion: 2.13.0
+InstallerType: exe
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: -q
+  SilentWithProgress: -q
+UpgradeBehavior: install
+ReleaseDate: 2023-07-12
+ElevationRequirement: elevationRequired
```

```suggestion
ElevationRequirement: elevationRequired
ExpectedReturnCodes:
  - InstallerReturnCode: 83
    ReturnResponse: missingDependency
    ReturnResponseUrl: https://www.ej-technologies.com/resources/install4j/help/doc/installers/errors.html
```

I guess this is how we can do it since 83 is the missing dependency exit code
Also linking https://github.com/microsoft/winget-cli/issues/360 since the InstallerType is Install4j